ksql/api.py

```diff
@@ -220,7 +223,7 @@ def inserts_stream(self, stream_name, rows):
 
         return return_arr
 
-    @staticmethod
+    @ staticmethod
```

Comment:
I'm not sure if we have to follow any coding style now. It's rare to see that there is a space after a decorator (`@`)

Reply:
that was my mistake! not sure how that happened